content/docs/reference-events.md

```diff
@@ -6,13 +6,13 @@ layout: docs
 category: Reference
 ---
 
-This reference guide documents the `SyntheticEvent` wrapper that forms part of React's Event System. See the [Handling Events](/docs/handling-events.html) guide to learn more.
+Este guia de referência documenta o agregador `SyntheticEvent` (evento sintético), que faz parte do sistema de eventos do React. Veja o guia [Manipulando Eventos (Handling Events)](/docs/handling-events.html) para saber mais.
 
-## Overview {#overview}
+## Visão geral {#visao-geral}
 
-Your event handlers will be passed instances of `SyntheticEvent`, a cross-browser wrapper around the browser's native event. It has the same interface as the browser's native event, including `stopPropagation()` and `preventDefault()`, except the events work identically across all browsers.
+Os manipuladores de evento (_event handlers_) serão passados como instâncias do `SyntheticEvent`, um agregador _cross-browser_ que envolve os eventos nativos do navegador. Ambos tem a mesma interface, incluindo `stopPropagation()` e `preventDefault()`, considerando que os eventos funcionem de forma idêntica entre os navegadores.
```

Comment:
```suggestion
Os manipuladores de evento (_event handlers_) serão passados como instâncias do `SyntheticEvent`, um agregador _cross-browser_ que envolve os eventos nativos do navegador. Ambos tem a mesma interface, incluindo `stopPropagation()` e `preventDefault()`, porém funcionam de forma idêntica em todos os navegadores.
```

I'm kinda uncomfortable with **wrapper** -> **agregador** and **wrap** -> **envolve**.

It it's an **agregador** it should **agregar**. Don't you think?

Also, the last phrase was losing it semantics, it should have an adversative meaning: "it has the same interface of native events (which do not work identically cross-browsers) with the exception of working identically"

Reply:
yup sounds good